prep-html-syntax/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<body>
+  <head>
```

Comment:
Whoops! Why is the `head` element in the `body` element? The `head` and `body` elements should both be children of the `html` element.

Also, your favorite character information is beautiful and horrible, great work :joy:

Reply:
Thank you! lol. Fixed the tags :)